package/MDAnalysis/core/AtomGroup.py

```diff
@@ -457,7 +457,84 @@
 # And the return route
 _SINGULAR_PROPERTIES = {v: k for k, v in _PLURAL_PROPERTIES.items()}
 
-_FIFTEEN_DEPRECATION = "This will be removed in version 0.15.0"
+_SIXTEEN_DEPRECATION = "This will be removed in version 0.16.0"
```

Comment:
The reason this went from "FIFTEEN" to "SIXTEEN" was because we were originally planning for the topology refactor to land in 0.15.0, with the release we're working toward now being a point release (0.14.1). Since we decided to release next as 0.15.0, this moved the topology refactor merge to 0.16.0.

Does that clarify things? I realize that it would be nice to have a more formal process for managing deprecations, but I think a major internal refactor like this is a rarity and this is about as clean as it probably gets.